src/containers/SignUpScreen/index.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  ImageBackground,
+  ScrollView
+} from 'react-native';
+import { connect } from 'react-redux';
+import { KeyboardAwareView } from 'react-native-keyboard-aware-view';
+import { object, func } from 'prop-types';
+
+import SignUpForm from '../../components/SignUpForm';
+import { signUp } from '../../actions/userActions';
+import styles from './styles';
+
+class SignUpScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.signUp = this.signUp.bind(this);
+    this.navigateToLogin = this.navigateToLogin.bind(this);
+  }
+
+  signUp(userData) {
+    const jsUser = userData.toJS();
+    jsUser.gender = jsUser.gender.toLowerCase();
+    const user = {
+      user: jsUser
+    };
+    this.props.userSignUp(user);
+  }
+
+  navigateToLogin() {
+    this.props.navigator.push({
+      screen: 'targetmobile.LoginScreen',
+      backButtonHidden: true
+    });
+  }
+
+  render() {
+    return (
+      <KeyboardAwareView animated>
+        <ScrollView>
+          <ImageBackground source={require('../../assets/images/groupTop.png')} style={styles.backImage}>
+            <View style={styles.content}>
+              <Text style={styles.title}>TARGET MVD</Text>
+              <View>
```

Comment:
is this `View` really necessary? you are not assigning any style to it and the form component inside probably has a `View` as it's outermost component. So any styling could be done there if necessary.

Reply:
Yeah, you're right. I suppose I added it there to organize myself and then I didn't delete it even though it didn't have any use.